### ISSUE TITLE
River Settings - Channels

### ISSUE THREAD

**Issue by 69mb** [ushahidi/SwiftRiver#127]
Implement the new river settings UI for adding channels.

![](http://dl.dropbox.com/u/48291217/Screenshots/k3hm.png)

**Comment by 69mb** [comment]
- Add channels in a popover window instead of the modal below. The modal is confusing.
- Also like in the mockup, say "Add feed" for rss and "Add parameter" only for Twitter. Add parameter is confusing for RSS.

![](http://o7.no/JVyH3u)
- When a parameter is saved, add a prompt asking the user if they want to add more channels.

**Comment by 69mb** [comment]
When a parameter is removed, remove drops from the river that were added by that parameter. Link drops to channel filters somehow?

**Comment by ekala** [comment]
Yeah, looks like we'll have to do that. Looks like the service stack (core) will also require some modifications to accommodate this no?

**Comment by ekala** [comment]
This was revised and implemented in the new UI prototype. See `integration-dev` branch.

@69mb  close?
